src/app.css

```diff
@@ -12,3 +12,7 @@
 .btn-primary:hover {
   background-color: #385a7f;
 }
+
+.pr-4-5 {
+  padding-right: 2.25rem;
+}
```

Comment:
Let's add this to `translator.css` instead and give it a specific name rather than `pr-4-5`. Perhaps `.translation-text-input`.